sql/tsql/TSqlParser.g4

```diff
@@ -1569,7 +1569,7 @@ alter_server_role_pdw
 
 // https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-service-transact-sql
 alter_service
-    : ALTER SERVICE modified_service_name=id_ (ON QUEUE (schema_name=id_ DOT) queue_name=id_)? (COMMA? (ADD|DROP) modified_contract_name=id_)*
+    : ALTER SERVICE modified_service_name=id_ (ON QUEUE (schema_name=id_ DOT)? queue_name=id_)?  ('(' (COMMA? (ADD|DROP) CONTRACT modified_contract_name=id_)* ')')?
```

Comment:
I believe this rule accepts a leading comma:

```
ALTER SERVICE [//Adventure-Works.com/Expenses] ( , ADD CONTRACT [//Adventure-Works.com/Expenses/ExpenseSubmission] ) 
                                                ^^^
```

Could you please fix it?

Reply:
ok i fixed it.